portal/templates/edit.html

```diff
@@ -0,0 +1,25 @@
+{% extends "layouts/base.html" %}
+
+{% block title %}Course Edit{% endblock %}
+
+{% block content %}
+<form method='post'>
+  <label for="name">
+    <input type = "text" name = "course-name" id='name' />
+    </label>
+    <label for="major">
+      <select name="class-major" id='major'>
+        <option value="cset" selected>CSET</option>
+        <option value="gen-ed">GEN-ED</option>
+        <option value="masonry">Masonry</option>
+        </select>
+      </label>
+      <label for="code">
+        <input type = "text" name = "course-code" id='code' />
+        </label>
+        <label for="description">
+        <input type = "text" name = "course-description" id='description'/>
+        </label>
+        <input type = "submit" />
```

Comment:
The indentation in this file is getting indented in areas it shouldn't be and it becomes clear on this line.